includes/ks/icon-row.html

```diff
@@ -0,0 +1,101 @@
+﻿<div class="content-row bg-umkc-gray">
+    <h2 style="margin-bottom:2em;">Icon Row (3 Up)</h2>
+    <div class="icon-row icon-row--small-stats jc-evenly">
```

Comment:
Seems like there are repeated groups -- Not sure how the styles are affected certain things.